File: 1-Iniciante/1008-Salario-2/1008.cs
using System;

public class HelloWorld
{
    public static void Main(string[] args)
    {
        var n1 = Console.ReadLine();
        
        double n2 = double.Parse(Console.ReadLine());
        double n3 = double.Parse(Console.ReadLine());
        
        Console.WriteLine ($"NUMBER = {n1}");
        Console.WriteLine ($"SALARY = U$ {n2*n3:f2}");
    }
}

File: 1-iniciante/1008.c
#include <stdio.h>
// 
int main(){
    double x[3];
    for(int i=0;i<3;++i){
        scanf("%lf",&x[i]);
    }
    printf("NUMBER = %.0lf\n",x[0]);
    printf("SALARY = U$ %.2lf\n",x[1]*x[2]);
    return 0;
}